Makefile

```diff
@@ -136,6 +139,25 @@ restart-control-plane-components:
 	kubectl -n numaflow-system delete po -lapp.kubernetes.io/component=numaflow-ux,app.kubernetes.io/part-of=numaflow --ignore-not-found=true
 	kubectl -n numaflow-system delete po -lapp.kubernetes.io/component=numaflow-webhook,app.kubernetes.io/part-of=numaflow --ignore-not-found=true
 
+.PHONY: deploy-nats
```

Comment:
nit: most of our e2e tests don't use nats and kafka, this change could add latency to those tests.

Reply:
Latency should be minimal since it is only introduced during the testing of one component, and each component is run in parallel. I'll give it some thought and create a separate PR if it can be avoided.